cypress/integration/trakt/comments.spec.js

```diff
@@ -0,0 +1,170 @@
+describe('04: Updates, reply and likes on comments.', () => { 
+    before(function(){
+        cy.getAccessToken();

```

Comment:
Indentation is required to be consistent for every new code line.  If it's 4 spaces, use 4 spaces for every new line in the code that belongs to the parent block.